src/routes/search/+page.svelte

```diff
@@ -0,0 +1,111 @@
+<script lang="ts">
+  import "./search-page.scss";
+  import { onMount } from "svelte";
+  import algoliasearch from "algoliasearch";
+  import instantsearch from "instantsearch.js";
+  import { configure, hits, searchBox, stats, pagination } from "instantsearch.js/es/widgets";
+  import { searchHitsTemplate } from "./searchHelpers";
+
+  const ALGOLIA_APP_ID = "9IIDYROXZ5";
+  const ALGOLIA_API_KEY = "0eac885faaa4c70dfc9dd8b6ab4ab10f";
+  onMount(() => {
```

Comment:
I mentioned this in the PR description, but given that all of the search elements are injected into the page here, it probably makes sense to include a loading spinner/indicator for potentially slower loads

Reply:
> Yeah, a loading indicator would be useful. Doesn't necessarily need to happen in this PR if you want to write a follow-up ticket for it.

I grabbed one from [here](https://projects.lukehaas.me/css-loaders/) as a quick addition. Worth running by @getpunched but should be fine as a placeholder, at least

Reply:
yeah, he's down with a [smaller color update](https://adhoc.slack.com/archives/C04EL8QK5FU/p1689362806331569). Those are applied [here](https://github.com/la-ldaf/ldaf-site/pull/212/commits/1202edd0eceb5d6b2192ffe046f43d0eb2328f47).